Commands/Provisioning/Site/AddDataRowsToProvisioningTemplate.cs

```diff
@@ -211,6 +210,7 @@ protected override void ExecuteCmdlet()
                 XMLTemplateProvider provider = new XMLFileSystemTemplateProvider(Path, "");
                 provider.SaveAs(template, Path, formatter, TemplateProviderExtensions);
             }
+            WriteObject(template);
```

Comment:
What is the reason you return the template object here? This changes the behaviour of the cmdlet as such, which is something we should not do.

Reply:
Hm the existing unit tests I was working on expected this return object. So I thought returning the object fell prey to a recent refactoring. If this assumption is not correct then the unit tests need to be changed.

Reply:
Like here:

https://github.com/SharePoint/PnP-PowerShell/blob/f05a9fcb6f9e29d7d159ca2c22eb735c331067b5/Tests/ListDataRowProvisioningTemplate.cs#L110

Reply:
I don't recall refactoring away the output of the cmdlet, as that would also mean I would have changed the behavior :-) Which we should not do. Please refactor/rewrite the unit tests to work without the return object. Thanks!

Reply:
For now I'll close this PR, awaiting a new PR.